src/pages/SignUp.jsx

```diff
@@ -186,27 +199,38 @@ export default function SignUp() {
                   />
                 </Grid>
                 <Grid item xs={12}>
-                  <TextField
-                    required
-                    fullWidth
-                    name="password"
-                    label="Password"
-                    id="password"
-                    autoComplete="new-password"
-                    type={showPassword ? 'text' : 'password'}
-                    onChange={handleUserDataChange('password')}
-                    endAdornment={
-                      <InputAdornment position="end">
-                        <IconButton
-                          aria-label="toggle password visibility"
-                          onClick={handleClickShowPassword}
-                          onMouseDown={handleMouseDownPassword}
-                        >
-                          {showPassword ? <VisibilityOff /> : <Visibility />}
-                        </IconButton>
-                      </InputAdornment>
-                    }
-                  />
+                  <FormControl fullWidth variant="outlined" margin="normal">
```

Comment:
This form control component looks duplicated with the one in Login.jsx. Can you think of a way to deduplicate?

Reply:
Refactored this in my next PR